source/future.tex

```diff
@@ -98,20 +98,64 @@
 prefix \tcode{++} and \tcode{\dcr} expressions\iref{expr.pre.incr}
 of \tcode{volatile}-qualified arithmetic and pointer types are deprecated.
 
+\begin{example}
+\begin{codeblock}
+volatile int velociraptor;
+++velociraptor;                 // deprecated: is this a single memory operation, two, or a six-foot turkey?
+\end{codeblock}
+\end{example}
+
+
 \pnum
 Certain assignments
 where the left operand is a \tcode{volatile}-qualified non-class type
 are deprecated; see~\ref{expr.ass}.
 
+\begin{example}
+\begin{codeblock}
+int neck, tail;
+volatile int brachiosaur;
+brachiosaur = neck;             // OK, a volatile store
+tail = brachiosaur;             // OK, a volatile load
+tail = brachiosaur = neck;      // deprecated: does this access brachiosaur once or twice?
+brachiosaur += neck;            // deprecated: does this access brachiosaur once or twice?
+brachiosaur = brachiosaur + neck; // OK, a volatile load, an addition, a volatile store
```

Comment:
These comments need to be aligned at a multiple of 4, among other things. See https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#code-blocks.